.github/workflows/modules-zstd.yml

```diff
@@ -75,6 +75,16 @@ jobs:
             bazel run -c opt $target -- --logtostderr;
           done
 
+      - name: Build and run verilog simulation of the ZSTD module components (opt)
+        if: ${{ !cancelled() }}
+        run: |
+          for target in $(bazel query 'filter(".*_cocotb_test$", kind(rule, //xls/modules/zstd/...))');
+          do
+            echo "running $target";
+            bazel run -c opt $target;
```

Comment:
should we use bazel test here, that would also allow us to get rid of the for loop.

Reply:
Rebased and changed the bazel call from `bazel run` to `bazel test`. This way we indeed got rid of the for loop.
We can still run those targets with `bazel run` if need be.